packages/unlock-js/src/walletService.ts

```diff
@@ -891,4 +891,30 @@ export default class WalletService extends UnlockService {
       callback
     )
   }
+
+  /**
+   * Update referrer fee
+   * @param {*} params
+   * @param {*} callback
+   */
+  async setReferrerFee(
+    params: {
+      lockAddress: string
+      address: string
+      feeBasisPoint: number
```

Comment:
same here. Instead of basis point, does it make sense to use percentage?

Reply:
maybe it causes confusion in this way, the value stores the basis point